### PR TITLE
GUVNOR-2452: adding new dependencies needed by new exec server ui

### PR DESCRIPTION
**Pull request by porcelli**

### CODE REVIEW

**Comment by manstis** [comment]
Jenkins please retest this.

File: kie-wb/kie-wb-webapp/pom.xml [hunk 1]
```diff
@@ -1596,6 +1607,11 @@
             <!-- drools-compiler has dependency on org.eclipse.jdt.core.compiler:ecj:jar:3.5.1:compile, see http://code.google.com/p/google-web-toolkit/issues/detail?id=4479 -->
             <gwtSdkFirstInClasspath>true</gwtSdkFirstInClasspath>
             <compileSourcesArtifacts>
+              <!-- Kie Server needed by Exec Server UI -->
+              <compileSourcesArtifact>org.kie:kie-api</compileSourcesArtifact>
+              <compileSourcesArtifact>org.kie.server:kie-server-api</compileSourcesArtifact>
+              <compileSourcesArtifact>org.kie.server:kie-server-controller-api</compileSourcesArtifact>
```
Comment (manstis) [inline kie-wb/kie-wb-webapp/pom.xml]: Do you not also need to add (some of) these to kie-wb's ```gwt.xml``` file?

Comment (porcelli) [inline kie-wb/kie-wb-webapp/pom.xml]: nope, as they are already declared in server management client gwt.xml

**Comment by manstis** [comment]
+1 to merge (when green)

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by csadilek** [comment]
Hm there are GWT compilation errors here which look like real given that the branch names line up: 

```
sources.jar!/org/kie/workbench/common/screens/server/management/service/SpecManagementService.java'
13:32:25 [INFO]          [ERROR] Line 65: No source code is available for type org.kie.server.controller.api.model.spec.ServerConfig; did you forget to inherit a required module?```

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by porcelli** [comment]
@csadilek i forgot to add dependencies to drools-wb-webapp =/

**Comment by csadilek** [comment]
ok that fixed the compilation error. There's a selenium test failure now related to the "server management browser". Could be this css selector needs an update? 

https://github.com/droolsjbpm/kie-wb-distributions/blob/master/kie-wb-smoke-tests/kie-wb-smoke-tests-main/src/main/java/org/kie/smoke/wb/selenium/model/persps/RuleDeploymentsPerspective.java#L24

https://kie-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/kie-wb-distributions-pullrequests-6.x/lastCompletedBuild/testReport/org.kie.smoke.wb.selenium.ui/LoadAllPerspectivesIntegrationTest/checkPerspectiveLoaded_Rule_Deployments_/

**Comment by porcelli** [comment]
@csadilek on it!

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by porcelli** [comment]
Jenkins please retest this.

**Comment by csadilek** [comment]
The failing tests have been fixed. Remaining failures are intermittent and not related to this work.

**Comment by csadilek** [comment]
6.4.x: 786620c8fa3f6e4b8a020bc2243f8318e3cdbfd2

**Pull request closed without merging**